lib/react_test_utils.dart

```diff
@@ -339,21 +355,41 @@ JsObject scryRenderedComponentsWithType(
 /// Finds all instances of components in the rendered tree that are DOM
 /// components with the class name matching className.
 List scryRenderedDOMComponentsWithClass(JsObject tree, String className) {
-  return _TestUtils.callMethod(
+  var components = _TestUtils.callMethod(
       'scryRenderedDOMComponentsWithClass', [tree, className]);
+
+  var jsObjectComponents = [];
+
+  components.forEach((component) {
+    if (component is JsObject) jsObjectComponents.add(component);
+    else jsObjectComponents.add(new JsObject.fromBrowserObject(component));
+  });
+
+  return jsObjectComponents;
 }
 
 /// Finds all instances of components in the rendered tree that are DOM
 /// components with the tag name matching tagName.
-JsObject scryRenderedDOMComponentsWithTag(JsObject tree, String tagName) {
-  return _TestUtils.callMethod(
+List scryRenderedDOMComponentsWithTag(JsObject tree, String tagName) {
+  var components = _TestUtils.callMethod(
       'scryRenderedDOMComponentsWithTag', [tree, tagName]);
+
+  var jsObjectComponents = [];
+
+  components.forEach((component) {
+    if (component is JsObject) jsObjectComponents.add(component);
+    else jsObjectComponents.add(new JsObject.fromBrowserObject(component));
+  });
```

Comment:
Yeah, I'd imagine that such a util could be useful for external use as well.

Something like:
```dart
JsObject normalizeReactComponent(dynamic component) {
  if (component is JsObject) return component;
  if (component is Element) return new JsObject.fromBrowserObject(component);
  if (element component == null) return null;

  throw new ArgumentError(component, 'component', 'not a valid ReactComponent');
}
```
Then, you can do:
```dart
components = components.map(normalizeReactComponent).toList();
```